tests/system/test_database_api.py

```diff
@@ -21,13 +21,16 @@
 from google.iam.v1 import policy_pb2
 from google.cloud import spanner_v1
 from google.cloud.spanner_v1.pool import FixedSizePool, PingingPool
+from google.cloud.spanner_admin_database_v1 import DatabaseDialect
 from google.type import expr_pb2
 from . import _helpers
 from . import _sample_data
-from google.cloud.spanner_admin_database_v1 import DatabaseDialect
 
 
 DBAPI_OPERATION_TIMEOUT = 240  # seconds
+FKDCA_CUSTOMERS_COLUMNS = ("CustomerId", "CustomerName")
```

Comment:
nit: Some places we are using `FKADC` and other places we are using `FKDCA` . It will be good to use same terminology everywhere.